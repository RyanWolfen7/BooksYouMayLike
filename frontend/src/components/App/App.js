import React from 'react'
import BooksList from '../BooksList/BooksList'

const App = ({ children }) => {
    return <BooksList/>
}

export default App