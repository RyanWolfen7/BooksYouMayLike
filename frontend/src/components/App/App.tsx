import React from 'react';

const App: React.FC = ({ children }) => {
  return (<h1>hello world</h1>
  );
}

export default App;
