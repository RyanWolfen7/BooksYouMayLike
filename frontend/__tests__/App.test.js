import React from 'react'
import { shallow } from 'enzyme'
import App from '../src/components/App/App'

let app

beforeAll(() => {
    app = shallow(<App />)
})

test('App to match snapshot', () => {
    expect(app).toMatchSnapshot()
})