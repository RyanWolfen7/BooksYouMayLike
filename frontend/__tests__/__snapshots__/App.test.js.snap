// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App to match snapshot 1`] = `
<WithStyles(ForwardRef(Grid))
  className="makeStyles-root-1"
  container={true}
  spacing={2}
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    xs={6}
  />
  <WithWidth(BooksList)
    handleLeftRender={[Function]}
  />
</WithStyles(ForwardRef(Grid))>
`;
